static/oas/v1/leagues/leagues.yaml

```diff
@@ -15,6 +15,27 @@ paths:
                 $ref: '#/schemas/GetLeaguesResponse'
         400:
           $ref: "../common/errors.yml#/responses/BadRequest"
+    post:
+      tags:
+        - Leagues
+      summary: Create a New League
```

Comment:
This is more of a picky comment, but I think you should have the summary be `summary: Create a League`. They should know it is new. This is the same with line 22.